pkg/policy/api/l4.go

```diff
@@ -49,15 +49,17 @@ type PortProtocol struct {
 	// +kubebuilder:validation:Optional
 	EndPort int32 `json:"endPort,omitempty"`
 
-	// Protocol is the L4 protocol. If omitted or empty, any protocol
-	// matches. Accepted values: "TCP", "UDP", "SCTP", "ANY"
+	// Protocol is the L4 protocol. If omitted or empty, any protocols
+	// with transport ports (TCP, UDP, SCTP) match.
```

Comment:
This is a bit wrong. "If omitted or empty, any protocol matches." is still correct. So this should be:

> Protocol is the L4 protocol. If omitted or empty, any protocol matches. "ANY" matches any protocol with L4 ports ("TCP", "UDP", "SCTP").